src/OSPSuite.InstallationValidator.Core/Presentation/MainPresenter.cs

```diff
@@ -11,17 +12,20 @@ public interface IMainPresenter : IDisposablePresenter
    {
       void SelectOutputFolder();
       void Abort();
-      Task StartInstallationValidation();
```

Comment:
Reason I changed it was that the started process is already completely asynchronous. There is no need to use the task and cancellation token as I would only use that to call process.kill and process.dispose anyway